app/views/documents/show/_scheduling_form.html.erb

```diff
@@ -5,12 +5,18 @@
 <% selected_time = submitted_values&.fetch("time") ||  current_scheduled_publish_datetime&.strftime("%l:%M%P")&.strip %>
 
 <%= form_tag(save_scheduled_publishing_datetime_path(@edition.document)) do %>
+
+  <% sheduleding_title = capture do %>
+    <% if current_scheduled_publish_datetime %>
+      Publish date: <br><%= current_scheduled_publish_datetime.strftime("%-d %B %Y - %l:%M%P") %>
+    <% else %>
+      Schedule publishing
+    <% end %>
+  <% end %>
+
   <%= render "govuk_publishing_components/components/details", {
-    title: if current_scheduled_publish_datetime
-             simple_format("Publish date: \n" + current_scheduled_publish_datetime.strftime("%-d %B %Y - %l:%M%P"))
```

Comment:
`simple_format` works fine, but there's no way to remove the wrapper, only to change the element type